docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md

```diff
@@ -0,0 +1,21 @@
+# Our maintenance plan
+
+This roadmap outlines responsibilities for administration of funds in the DCAF-administered Heroku pipeline (e.g. where DCAF administers DARIA for them), and roadmapping new development.
+
+## Administration
+
+* The engineering team has set up a series of monitors for applications, and is responsible for keeping an eye on them and troubleshooting when something goes wrong.
```

Comment:
Which applications? All deployed applications, specific Funds applications?

Reply:
Probably just easier to link to https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SECURITY.md

Reply:
I think 677822e5 addresses @lomky 's point here!